dts/bindings/serial/zephyr,cdc-acm-uart.yaml

```diff
@@ -8,3 +8,18 @@ compatible: "zephyr,cdc-acm-uart"
 include: uart-controller.yaml
 
 on-bus: usb
+
+properties:
+    tx-fifo-size:
+      type: int
+      default: 1024
+      description: |
+        Size of the virtual CDC ACM UART TX FIFO
```

Comment:
You must justify why you chose both of these defaults in the description field here

Reply:
I do not have good reason to justify it in the description, it does not make sense without knowing development history of CDC ACM UART implementation. But I mentioned it in the commit message, 
"...New properties takes the default
size value of USB_CDC_ACM_RINGBUF_SIZE which can be removed
subsequently."